src/ssl.c

```diff
@@ -54519,12 +55064,13 @@ WOLFSSL_STRING *wolfSSL_TXT_DB_get_by_index(WOLFSSL_TXT_DB *db, int idx,
     }
     return (WOLFSSL_STRING*) wolfSSL_lh_retrieve(db->data, value);
 }
-#endif
 
 /*******************************************************************************
  * END OF TXT_DB API
  ******************************************************************************/
 
+#endif
```

Comment:
Move #endif back.
The #ifdef is inside "START OF"

Reply:
Moved.